app/ports/management/commands/update-portinfo.py

```diff
@@ -0,0 +1,50 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from parsing_scripts import git_update, rsync_operations
+from ports.models import Port, LastPortIndexUpdate
+
+
+class Command(BaseCommand):
+    help = "Populates the database with Initial data from portindex.json file"
+
+    def add_arguments(self, parser):
+        parser.add_argument('new',
+                            nargs='?',
+                            default=False,
+                            help="Define a commit till which the update should be processed")
+        parser.add_argument('old',
+                            nargs='?',
+                            default=False,
+                            help="Not recommended. Helps you provide a commit from which update should start")
+
+    def handle(self, *args, **options):
+        # Fetch from rsync
```

Comment:
https://github.com/macports/macports-webapp/pull/33#discussion_r302764260 I have added comments for steps to make the logic a bit more clear.